code/drasil-data/lib/Data/Drasil/Concepts/Documentation.hs

```diff
@@ -47,9 +47,9 @@ doccon = [abbAcc, abbreviation, acronym, analysis, appendix, aspect, body, charO
 -- | Collects all documentation-related common ideas (like a concept, but with no definition).
 doccon' :: [CI]
 doccon' = [assumption, dataConst, dataDefn, desSpec, genDefn, goalStmt, inModel,
-  likelyChg, mg, mis, notApp, physSyst, requirement, srs, thModel, typUnc, unlikelyChg, notebook, refBy, refName]
```

Comment:
Why doesn't this affect the changes from #3389? "Refname" and "Refby" still appear in Projectile's SRS Table of Acronyms....

Reply:
Didn't the changes from #3389 added them to the table?

Reply:
Interesting. It added them to `doccon'` but also the domain variants of them to `srsDomains`, which might be where they are actually used from. This doesn't seem to be an issue with this PR.

https://github.com/JacquesCarette/Drasil/blob/ed132e8d3373e336668a2bc5f2a19f93847b968e/code/drasil-data/lib/Data/Drasil/Concepts/Documentation.hs#L49-L50
https://github.com/JacquesCarette/Drasil/blob/ed132e8d3373e336668a2bc5f2a19f93847b968e/code/drasil-data/lib/Data/Drasil/Concepts/Documentation.hs#L336-L337